### PR TITLE
PSY -> PSSE Exporter

### PR DESCRIPTION
**Pull request by HaleyRoss**
Adding raw code for a PowerSystems -> PSS/e exporter module, plus the foundation of the testing for the new exporter. To be cleaned, reformatted, and integrated into the rest of the code base still.

### CODE REVIEW

File: src/psse_exporter/compare_psse_results.jl
Comment (jd-lara) [inline src/psse_exporter/compare_psse_results.jl:1]: Why is this here?

Comment (scdhulipala) [reply src/psse_exporter/compare_psse_results.jl:1]: This hasn't been cleaned up yet. I will make this a PR after we've cleaned it up and is ready for your review.

**Comment by GabrielKS** [comment]
Closing as superseded by the "stable" branch https://github.com/NREL-Sienna/PowerFlows.jl/tree/hrgks/psse_exporter_psy4, the "development" branch https://github.com/NREL-Sienna/PowerFlows.jl/tree/gks/hr/formalize_psse_exporter, and the draft PR https://github.com/NREL-Sienna/PowerFlows.jl/pull/42.
